js-tests/build/specs.js

```diff
@@ -190,6 +190,12 @@ var wp = (typeof window !== "undefined" ? window['wp'] : typeof global !== "unde
 
 describe( "MCE View Constructor", function() {
 
+	beforeEach( function() {
+		wp.shortcode.regexp = function( tag ) {
+			return new RegExp( '\\[(\\[?)(' + tag + ')(?![\\w-])([^\\]\\/]*(?:\\/(?!\\])[^\\]\\/]*)*?)(?:(\\/)\\]|\\](?:([^\\[]*(?:\\[(?!\\/\\2\\])[^\\[]*)*)(\\[\\/\\2\\]))?)(\\]?)', 'g' );
```

Comment:
Rather than copy and paste this regex, should our copy in `js-tests/vendor/wp-shortcode.js` be the unmemoized version?